example_inputs.yaml

```diff
@@ -0,0 +1,111 @@
+{
+    "R1_output_fastq": "processed_fastq_R1.fastq.gz",
+    "R2_output_fastq": "processed_fastq_R2.fastq.gz",
+    "abra2_bam_index": true,
+    "abra2_consensus_sequence": null,
+    "abra2_contig_anchor": null,
+    "abra2_maximum_average_depth": null,
+    "abra2_maximum_mixmatch_rate": null,
+    "abra2_no_edge_complex_indel": true,
+    "abra2_no_sort": null,
+    "abra2_output_bams": "test_abra.bam",
+    "abra2_scoring_gap_alignments": null,
+    "abra2_soft_clip_contig": null,
+    "abra2_window_size": null,
+    "apply_bqsr_output_file_name": null,
+    "base_recalibrator_output_file_name": null,
+    "bedtools_genomecov_option_bedgraph": true,
+    "bedtools_merge_distance_between_features": null,
+    "bwa_mem_K": 1000000,
+    "bwa_mem_M": null,
+    "bwa_mem_P": null,
+    "bwa_mem_T": 30,
+    "bwa_mem_Y": true,
+    "bwa_mem_output": "test_alignment.sam",
+    "bwa_number_of_threads": null,
+    "comment": null,
+    "create_bam_index": true,
+    "description": null,
+    "duplicate_scoring_strategy": null,
+    "fastp_failed_reads_output_file_name": null,
+    "fastp_html_output_file_name": "test_fastp_out.html",
+    "fastp_json_output_file_name": "test_fastp_out.json",
+    "fastp_minimum_read_length": 25,
+    "fastp_read1_adapter_sequence": "GATCGGAAGAGC",
+    "fastp_read1_output_file_name": "trimmed_fastp_R1.fastq.gz",
+    "fastp_read2_adapter_sequence": "AGATCGGAAGAGC",
+    "fastp_read2_output_file_name": "trimmed_fastp_R2.fastq.gz",
+    "fastp_unpaired1_output_file_name": null,
+    "fastp_unpaired2_output_file_name": null,
+    "fgbio_fastq_to_bam_input": [
+[
+        {
+            "class": "File",
+            "path": "/path/to/Sample_R1_part1.fastq.gz"
+        },
+        {
+            "class": "File",
+            "path": "/path/to/Sample_R2_part1.fastq.gz"
+        }
+],
+[
+        {
+            "class": "File",
+            "path": "/path/to/Sample_R1_part2.fastq.gz"
+        },
+        {
+            "class": "File",
+            "path": "/path/to/Sample_R2_part2.fastq.gz"
+        }
+]
+    ],
```

Comment:
I know I suggested to do it this way, but we could also consider having separate inputs for R1 and R2 fastqs instead of relying on the sort order of these arrays. 

Either that, or we should document that R1 should come before R2, and perhaps rename this input to `fgbio_fastq_to_bam_paired_fastqs`, or something more informative of the structure here

Not a huge deal though

Reply:
Thanks, we can list it in the inputs